Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/FileTransporter.cs

```diff
@@ -132,16 +132,18 @@ private bool CreateDestinationDirIfNeeded(out FileTransferResult fileTransferRes
     private static FtpClient CreateFtpClient(Connection connect)
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/4b84f8f4c7826a0aaeea37de412aae81837f5742/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/FileTransporter.cs#L92
No need to define the class here.